tests/App/Entity/Book.php

```diff
@@ -34,6 +34,7 @@ class Book
 
     /**
      * @ORM\ManyToOne(targetEntity="Author", inversedBy="books")
+     * @ORM\JoinColumn(name="author_id", referencedColumnName="id", onDelete="SET NULL")
```

Comment:
It was missing this line to avoid problems when deleting authors.

Reply:
Since the test was green without this, there is something to do in order to improve the test no ? Like checking there is no 'error' flash bag.

Reply:
Now it should be better.